atsign-foundation/at_gauges
# chore: Ready to Publish to pub.dev

**- What I did**

- Merged the at_radial_gauges and at_linear_gauges as one package.

- Prepared repo for publishing to pub.dev

**- How I did it**

**- How to verify it**

- Run the example app, you will see the SimpleRadialGauge, ScaleRadialGauge and the SimpleLinearGauge
- Open doc/api/index.html in the browser

**- Description for the changelog**

Merged the at_radial_gauges and at_linear_gauges as one package
